dev-support/database/init-database.sh

```diff
@@ -39,7 +39,6 @@ mysql -e "CREATE USER IF NOT EXISTS 'metastore'@'%' IDENTIFIED BY 'password';"
 mysql -e "GRANT ALL PRIVILEGES ON *.* TO 'metastore'@'%';"
 mysql -e "use metastore; source ./metastore.sql; show tables;"
 
-mysql -e "CREATE DATABASE IF NOT EXISTS mlflow;"
+mysql -e "CREATE DATABASE IF NOT EXISTS mlflowdb;"
 mysql -e "CREATE USER IF NOT EXISTS 'mlflow'@'%' IDENTIFIED BY 'password';"
 mysql -e "GRANT ALL PRIVILEGES ON *.* TO 'mlflow'@'%';"
-mysql -e "use mlflow; source ./mlflow.sql; show tables;"
```

Comment:
Why this line is removed? Is it only for debugging purposes?

Reply:
Because mlflow server would automatically create table by itself.